api/src/server.ts

```diff
@@ -0,0 +1,133 @@
+/* eslint @typescript-eslint/no-var-requires: "off"
```

Comment:
I think this is another annotation that is no longer needed.

Reply:
made change